src/Telemetry/TelemetryService.php

```diff
@@ -42,7 +42,7 @@ public static function isTelemetryEnabled(): int
     {
         // Check if telemetry is disabled in the product registration table.
         $isEnabled = sqlQuery("SELECT `telemetry_disabled` FROM `product_registration` WHERE `telemetry_disabled` = 0")['telemetry_disabled'] ?? null;
-        if ((int)$isEnabled === 0) {
+        if (!is_null($isEnabled)) {
```

Comment:
@sjpadgett , This was returning 1 no matter what. I am guessing that forcing null to into makes it a zero. The above change works nicely on testing (when no entry, when null, when 0, and when 1).

Reply:
yeah I should know better. lol I just told Ruth about this!